src/core/server/mocks.ts

```diff
@@ -37,6 +37,7 @@ export { elasticsearchServiceMock } from './elasticsearch/elasticsearch_service.
 export { httpServiceMock } from './http/http_service.mock';
 export { loggingServiceMock } from './logging/logging_service.mock';
 export { savedObjectsClientMock } from './saved_objects/service/saved_objects_client.mock';
+export { savedObjectsRepositoryMock } from './saved_objects/service/lib/repository.mock';
```

Comment:
Task Manager interacts with the repository directly, so accessing this mock was needed for testing purposes.

Reply:
it's okay. The SO repository is a part of public contract https://github.com/elastic/kibana/blob/7ffe38569e48cbebfeb00bd084799d8aa48f38f2/src/core/server/saved_objects/saved_objects_service.ts#L260